docs/docs/api/06-Configuration/askui-ui-control-client.md

```diff
@@ -3,15 +3,34 @@ custom_edit_url: null
 ---
 # askui UI Control Client
 
-## Table of Contents
-
-  - [Properties](#properties)
-    - [uiControllerUrl](#uiControllerUrl)
-    - [inferenceClientUrl](#inferenceClientUrl)
-    - [annotationLevel](#annotationLevel)
-    - [credentials](#credentials)
-  - [Set Log Level](#set-log-level)  
-  - [Example](#example)  
```

Comment:
Like for removing it. Just saw that it basically duplicates the TOC on the right. But can you remove it in `askui-ui-controller.md` as well for consistency?